aea/manager.py

```diff
@@ -607,3 +664,53 @@ def _create_private_key(self, name, ledger) -> str:
         path = os.path.join(self._keys_dir, f"{name}_{ledger}_private.key")
         create_private_key(ledger, path)
         return path
+
+    def _load_state(self, local: bool) -> None:
+        """
+        Load saved state from file.
+
+        :param local: bool is local project and agents re-creation.
+
+        :return: None
+        :raises: ValueError if failed to load state.
+        """
+        if not os.path.exists(self._save_path):
+            return
+
+        save_json = {}
+        with open(self._save_path) as f:
+            save_json = json.load(f)
```

Comment:
can throw an error..handle it

Reply:
Which kind of error? If file doesn't exist it creates one, if exists - overwrites one.